package.json

```
@@ -31,7 +31,7 @@
     "ci-test": "jest",
     "format:write": "prettier --write **/*.js",
     "format:check": "prettier --check **/*.js",
-    "package": "ncc build src/index.js --license licenses.txt",
+    "package": "ncc build src/index.js --license LICENSE",
```

Why is this change needed? I don't think we have licenses.txt file.

---

The license file we have in the GitHub repo is called `LICENSE`, we indeed do not have a `licenses.txt`. Unless I'm misinterpreting the flag, this just adds that `LICENSE` file to the dist folder.

---

yes, lets revert this change

```suggestion
    "package": "ncc build src/index.js --license LICENSE",
```